tests/components/onewire/test_init.py

```diff
@@ -63,27 +62,6 @@ async def test_unload_entry(hass: HomeAssistant, config_entry: MockConfigEntry)
     assert config_entry.state is ConfigEntryState.NOT_LOADED
 
 
-async def test_update_options(
```

Comment:
Test is removed as there is no update listener anymore (this is not testing an options flow)